int32 num_1
int32 num_2
---
int32 sum
